plugins/catalog-react/src/components/EntityPeekAheadPopover/EntityPeekAheadPopover.tsx

```diff
@@ -90,27 +84,24 @@ export const EntityPeekAheadPopover = (props: EntityPeekAheadPopoverProps) => {
     variant: 'popover',
     popupId: 'entity-peek-ahead',
   });
-  const compoundEntityRef = parseEntityRef(entityRef);
   const [isHovered, setIsHovered] = useState(false);
 
-  const debouncedHandleMouseEnter = debounce(
-    () => setIsHovered(true),
-    delayTime,
+  const debouncedHandleMouseEnter = useMemo(
```

Comment:
memo'd since `debounce` is stateful and should not be recreated on every render

Reply:
thank you :)